wordpress-mobile/WordPress-Android
# Issue/10331 remove wpnumberpicker

Fixes #10331

This PR removes `WPNumberPicker`, since it's broken on devices with API 28+

We are switching back to default Number Picker, styled to look similar to the custom one.
`NumberPicker` is notoriously hard to customize without reflection, but it looks and works similar to the `WPNumberPicker`.


`NumberPicker` is on the left `WPNumberPicker` is on the right.

[![Image from Gyazo](https://i.gyazo.com/d0022d5de0995b89f2bf4d5d83cfe176.png)](https://gyazo.com/d0022d5de0995b89f2bf4d5d83cfe176)

To test:
On a device with API 28+, navigate to Site settings and check the following preferences, that utilize `NumberPicker`:
- Posts per page
- Threading
- Paging
Make sure they work as expected.


Update release notes:
